4-genotype_qc/1-compare_hard_filter_combinations.py

```diff
@@ -100,140 +106,123 @@ def prepare_giab_ht(giab_vcf: str, giab_cqfile: str, mtdir: str) -> hl.Table:
     return giab_vars
 
 
-def annotate_cq(mt: hl.MatrixTable, cqfile: str) -> hl.MatrixTable:
-    '''
-    Annotate MatrixTable with consequence
-    :param hl.MatrixTable mt: Input MatrixTable
-    :param str cqfile: Most severe consequence annotation from VEP
-    :return: hl.MatrixTable
-    '''
-    ht = hl.import_table(cqfile, types={'f1': 'int32'}, no_header=True)
-    ht = ht.rename({'f0': 'chr', 'f1': 'pos', 'f2': 'rs',
-                    'f3': 'ref', 'f4': 'alt', 'f5': 'consequence'})
-
-    ht = ht.key_by(
-        locus=hl.locus(ht.chr, ht.pos), alleles=[ht.ref, ht.alt]
-    )
-    ht = ht.drop(ht.chr, ht.pos, ht.ref, ht.alt)
-
-    mt = mt.annotate_rows(consequence=ht[mt.row_key].consequence)
-
-    return mt
-
-
-def filter_and_count(mt_path: str, ht_giab: hl.Table, pedfile: str, mtdir: str, config: dict) -> dict:
-    '''
+def filter_and_count(mt_path: str, ht_giab: hl.Table, pedfile: str, mtdir: str, conf: dict) -> dict:
```

Comment:
Do we plan to use the approach of **kwargs instead of passing config dict in these substeps for this refactoring?

Reply:
We have another task that covers the refactoring of this part: HSH-175. Here, I tried to introduce a minimum required amount of changes.